README.md

```diff
@@ -61,6 +61,10 @@ By default, the style-loader appends `<style>` elements to the end of the `<head
 
 If defined, the style-loader will re-use a single `<style>` element, instead of adding/removing individual elements for each required module. **Note:** this option is on by default in IE9, which has strict limitations on the number of style tags allowed on a page. You can enable or disable it with the singleton query parameter (`?singleton` or `?-singleton`).
 
+#### `tagAttrs`
+
+If defined, style-loader will attach given attributes with there values on `<style>` / `<link>` element.

```

Comment:
Can you also add what this option expects? Is it an array or object? And what do I exactly need to define in that?